proto/wtools/abase/l0/l1/5ArgumentsArray.s

```diff
@@ -287,7 +324,10 @@ var ArgumentsArrayExtension =
   // dichotomy
 
```

Comment:
where is `isOld` and `likeOld`?

Reply:
is and like are on original version. Should i duplicate functions and name them isOld and likeOld?

Reply:
Yes, please